src/main/java/teammates/common/datatransfer/attributes/InstructorAttributes.java

```diff
@@ -18,131 +18,59 @@
     public static final String DEFAULT_DISPLAY_NAME = "Instructor";
 
     // Note: be careful when changing these variables as their names are used in *.json files.
+
+    /** Required fields. */
     public String googleId;
     public String courseId;
     public String name;
     public String email;
-    public Boolean isArchived;
+
+    /** Optional fields. */
     public String key;
     public String role;
-    public boolean isDisplayedToStudents;
     public String displayedName;
-
+    public Boolean isArchived;
+    public boolean isDisplayedToStudents;
     public InstructorPrivileges privileges;
 
     /**
-     * Creates a new instructor with default access level and default displayedName.
+     * Return new builder instance with default values for optional fields.
      *
-     * @deprecated only to be used for testing
-     */
-    @Deprecated
-    public InstructorAttributes(String googleId, String courseId, String name, String email) {
-        this(googleId, courseId, name, email,
-             Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER, DEFAULT_DISPLAY_NAME,
-             new InstructorPrivileges(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER));
-    }
-
-    /**
-     * Creates a new instructor with params specified (isDisplayedToStudent is set to true by default).
-     */
-    public InstructorAttributes(String googleId, String courseId, String name, String email, String role,
-                                String displayedName, String instructorPrivilegesAsText) {
-        this.googleId = SanitizationHelper.sanitizeGoogleId(googleId);
-        this.courseId = SanitizationHelper.sanitizeTitle(courseId);
-        this.isArchived = false;
-        this.name = SanitizationHelper.sanitizeName(name);
-        this.email = email;
-        this.role = SanitizationHelper.sanitizeName(role);
-        this.isDisplayedToStudents = true;
-        this.displayedName = SanitizationHelper.sanitizeName(displayedName);
-        this.privileges = getInstructorPrivilegesFromText(instructorPrivilegesAsText);
-    }
-
-    /**
-     * Creates an instructor (isDisplayedToStudent is set to true by default).
+     * <p>Following default values are set to corresponding attributes:
+     * {@code false} for {@code isArchived}
+     * {@code true} for {@code isDisplayedForStudents}
+     * {@code DEFAULT_DISPLAYED_NAME} for {@code displayedName}
+     * {@code INSTRUCTOR_PERMISSION_ROLE_COOWNER} for {@code role}
+     * {@code new InstructorPrivileges(role)} for {@code privileges}
```

Comment:
Easier to writ like this?
```java
* {@code isArchived = false} <br>
```
Also, without the `<br>` this is going to appear as one line in the generated javadoc.